Dockerfile

```diff
@@ -1,6 +1,14 @@
 FROM registry.centos.org/centos/centos:7
 
 ENV F8A_WORKER_VERSION=c019a1e
+LABEL name="f8analytics backbone services" \
+      description="Stack aggregation and recommendation service." \
+      git-sha="46e443d" \
+      email-ids="nshaikh@redhat.com," \
+      git-url="https://github.com/fabric8-analytics/f8a-server-backbone" \
+      git-path="/" \
+      target-file="Dockerfile" \
+      app-license="GPL-3.0"
```

Comment:
using `app-license` instead of `license` field to ensure, the base image inherited Labels fields are not overridden.